db/query.go

```diff
@@ -238,11 +238,12 @@ func (context *DatabaseContext) N1QLQueryWithStats(queryName string, statement s
 
 	results, err = gocbBucket.Query(statement, params, consistency, adhoc)
 	if err != nil {
-		context.DbStats.StatsGsiViews().Add(fmt.Sprintf(base.StatKeyN1qlQueryErrorCountExpvarFormat, queryName), 1)
+		context.DbStats.NewStats.GSIStats(queryName).QueryErrorCount.Add(1)
 	}
 
-	context.DbStats.StatsGsiViews().Add(fmt.Sprintf(base.StatKeyN1qlQueryCountExpvarFormat, queryName), 1)
-	context.DbStats.StatsGsiViews().Add(fmt.Sprintf(base.StatKeyN1qlQueryTimeExpvarFormat, queryName), time.Since(startTime).Nanoseconds())
+	context.DbStats.NewStats.GSIStats(queryName).QueryErrorCount.Add(1)
```

Comment:
This should just be QueryCount.

Reply:
Good catch. This is something that I'll have to double check too, its easier to check things like this in the PR review so I'll make sure to do that!